src/Features/VisualBasic/Portable/RemoveUnusedVariable/VisualBasicRemoveUnusedVariableCodeFixProvider.vb

```diff
@@ -41,5 +41,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.RemoveUnusedVariable
         Protected Overrides Function GetVariables(localDeclarationStatement As LocalDeclarationStatementSyntax) As SeparatedSyntaxList(Of SyntaxNode)
             Return localDeclarationStatement.Declarators
         End Function
+
+        Protected Overrides Function GetCandidateLocalDeclarationForRemoval(declarator As ModifiedIdentifierSyntax) As LocalDeclarationStatementSyntax
+            Return Nothing
```

Comment:
Why is this returning Nothing?

Reply:
Can we add a unit test to validate this code path?

Reply:
Originally i thought that the code in the base class only was necessary for C# since it was clearly beeing hit and causing the original issue that this PR is for.

However i just debugged a bit and it seems to me we can get rid of `MergeNodesToRemove` in the base class altogether, since the subsequent `RemoveNode` code seems to deal with unmerged nodes just fine (which might originally not have been the case). If i comment out the complete body for `MergeNodesToRemove` i still get all green unit tests for both vb and c#. 

Should i keep it and maybe add a comment here in the vb implementation to signal why it returns nothing, or should i just get rid of the unnecessary code?

Reply:
I am not able to convince myself by looking at the code that `MergeNodesToRemove` is redundant. I also think the correct implementation for VB here should find the parenting local declaration statement, similar to the implementation in the previous file. What happens if you change the implementation to match that? Also can you confirm if existing tests are hitting this path?

Reply:
I can confirm that this path is beeing hit by existing tests.

> I am not able to convince myself by looking at the code that MergeNodesToRemove is redundant. I also think the correct implementation for VB here should find the parenting local declaration statement, similar to the implementation in the previous file.

will do.

Reply:
@mavasani Since the implementations are now matching between C# and VB i pulled the code up into the base class and got rid of the function altogether. Tests are still passing.